plugin.json

```diff
@@ -59,7 +59,7 @@
             "name": "pollInterval",
             "description": "A number in milisecond to define the poll interval.",
             "type": "number",
-            "default": 1000
+            "default": 5000

```

Comment:
What is the real cause to change this value to 5secs? If it is really needed to increase, we will have to get buy in from PMs for this new poll interval.

Reply:
The Json RPC response usualy takes more than 1-2 sec, and I observed there are connection refused failures in case there are multiple calls before the response is read from the socket. keeping 5 sec gives a decent performance. Do you want me to keep it 1 sec ?

Reply:
My only worry is, we should not be dealing with time just because something is not performing fast, what happens when Json RPC takes more than 5-6 secs now? I believe, we should be handling this on plugin side to wait before it polls again if old poll is currently active... what ya'think?

Reply:
The polling happens using framework DataSource poller and this would require some major rework in terms of framework. Lets keep it as 1 sec for now as it has been already tested this way.